character/src/ts/main.tsx

```diff
@@ -1,58 +1,54 @@
-// This Source Code Form is subject to the terms of the Mozilla Public
-// License, v. 2.0.If a copy of the MPL was not distributed with this
-// file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
-// Character Unit Frame
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
 
 /// <reference path="../tsd/tsd.d.ts" />
+
 import * as React from 'react';
-import * as Reflux from 'reflux';
+import * as cuAPI from 'cu-core';
```

Comment:
delete this line.  You're not using the import additionally, you should not import anything as cuAPI that's a reserved variable name for our client -> ui object.